### PR TITLE
chore: added GDG community link in communities section issue-964

### LINKED ISSUE #964: [Add] Community link in the community section.
### Site URL

https://gdg.community.dev/

### Description of the above provided site

GDG(Google Developer Group) is the best community organising multiple events, workshops for folks those who are related to the  tech community.
So adding this websites link in the project will be fair deal for beginners .
If this is required do assign it to me under GSSOC 2023 !
Changes will be done in no time.  : |)

### Category

Frontend

### Sub Category

link

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by ManishaSwain8**
Added GDG community link in communities section
## Fixes Issue
close #964 

Screenshot (69)

### CODE REVIEW

File: database/other/communities.json [hunk 1]
```diff
@@ -1 +1,9 @@
-[]
\ No newline at end of file
+[
+    {
+        "name": "Google Developer Groups",
+        "description": "It gives opportunity to learn new skills in a variety of formats. Here multiple events,workshops ,meet-ups also organised virtually or in-person with similar interests in technology",
```
Comment (CBID2) [inline database/other/communities.json:4]: [Proofreading]
```suggestion
        "description": "It gives tech professionals opportunity to learn new skills in a variety of formats. Here, events, workshops, meet-ups are organised virtually or in-person for people with similar interests in technology",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ManishaSwain8! :) I just made some suggestions on how to improve your description. 😄

File: database/other/communities.json (hunk 1 above)
Comment (ManishaSwain8) [reply database/other/communities.json:4]: Thanks for feedback, I am changing it definitely.

Comment (ManishaSwain8) [reply database/other/communities.json]: @CBID2 I have updated the description, do check it out  : |)

**Review by CBID2 — APPROVED** [review]
Well done! :) LGTM! :)

**Review by ujjawaltyagii — APPROVED** [review]
lgtm 👍

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**